chart/compass/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -82,6 +90,14 @@ spec:
                 secretKeyRef:
                   name: compass-postgresql
                   key: postgresql-sslMode
+            - name: APP_DEFAULT_TENANT
+              value: "{{ .Values.global.defaultTenant }}"
+            - name: APP_DIRECTOR_URL
+              value: "https://{{ .Values.global.gateway.tls.secure.oauth.host }}.{{ .Values.global.ingress.domainName }}/director/graphql"
```

Comment:
`https://compass-gateway-auth-oauth.kyma.local/director/graphql` is it correct URL?
what about port? where are `namespace` and `svc` parts?

Reply:
changed to `http://compass-director.compass-system.svc.cluster.local:3000/graphql`